src/angular/components/view.component.ts

```diff
@@ -6,6 +6,7 @@ import {
     OnDestroy,
     OnInit,
     Output,
+    Directive,
```

Comment:
linter should complain not alphabetial

Reply:
We seem to be missing the `ordered-imports` rule in tslint configs. I added the rule and ran `lint:fix`.

Reply:
After digging a bit, it seems that tslint:recommended has changed a bit between versions. Since it was updated during the `TypeScript` upgrade we got the new recommended rules.

Most if not all of the code formatting rules have been removed, https://github.com/palantir/tslint/pull/4871. TSlint is now also deprecated with the recommendation to migrate over to ESlint and use Prettier for code formatting rules.

We can either re-add the removed rules or migrate to ESlint and Prettier.